src/pybind/mgr/dashboard/rgw_detail.html

```diff
@@ -0,0 +1,101 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+<script>
+        $(document).ready(function(){
+            // Pre-populated initial data at page load
+            var content_data = {{ content_data }};
+
+            var refresh = function() {
+                $.get("{{ url_prefix }}/rgw/rgw_data/" + content_data.rgw_id + "/", function(data) {
+                   content_data = data;
+                   setTimeout(refresh, 3000);
+                });
+            };
+            setTimeout(refresh, 5000);
+
+            var post_load = function() {
+                content_data.rgw_metadata_list = [];
+                content_data.rgw_status_list = [];
+                _.each(content_data.rgw_metadata, function(v, k) {
+                    content_data.rgw_metadata_list.push({
+                        key: k,
+                        value: v
+                    });
+                });
+                _.each(content_data.rgw_status, function(v, k) {
+                    content_data.rgw_status_list.push({
+                        key: k,
+                        value: v
+                    });
+                });
+                
+                if (content_data.rgw_status_list.length() == 0) {
```

Comment:
You can use `rv-show="rgw_status_list | length"` in the markup, to avoid the need to explicitly set a _show variable up here (the `length` helper is defined in base.html so you can use it anywhere)